src/modules/chat_moderator_cards/style.css

```diff
@@ -0,0 +1,49 @@
+
+.bttv-font-size-1 {
+    font-size: 1rem;
+}
+
+button.bttv-font-size-1:focus {
+    color: inherit;
+}
+
+.bttv-hide-section {
+    display: none!important;
+}
+
+.viewer-card__overlay {
+    background: rgba(20, 20, 20, 0.7);
+}
+
+.bttv-stats, .bttv-created-at {
+    color: #fff;
+}
+
+.bttv-messages-scroll {
+    overflow-y: auto;
+    max-height: 250px;
+    width: 1000px;
+
+    &::-webkit-scrollbar {
+        width: 12px;
+    }
+
+    &::-webkit-scrollbar-thumb {
+        border-radius: 10px;
+        -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,.3);
+        background-color: #777;
+    }
+
+    .chat-line__mod-icons {
+        display: none!important;
+    }
+}
+
+[data-a-target=chat-message-mention] {
```

Comment:
this is a bad selector

Reply:
@night You mean the double quotes missing? Otherwise this selector is used in "doubleclick_mention" module, how is it bad?